index.js

```diff
@@ -92,7 +94,11 @@ async function run() {
         })
```

Comment:
I've never developed a Github action, so I wasn't sure if the `if` condition should also impact line 89, where the check conclusion is set. If so, this might look more like this:
```javascript
          conclusion: allowFailureInput ? 'success' : 'failure',
```

Reply:
Indeed I believe you also need to update the `conclusion` otherwise there may hang older checks with incorrect status